Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+
+    var description: String = ""
+    var timestamp: String = ""
+
+}
+
+enum Stage {
+
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+
+}
+
+// MARK: - Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
+    static func saveToKeychain(loginInfo: LoginInfo) {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = loginInfo.netid as AnyObject
+        keychain["password"] = loginInfo.password as AnyObject
+    }
+    
+    static func removeKeychainLoginInfo() {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = nil
+        keychain["password"] = nil
+    }
+    
+    static func loadFromKeychain() -> LoginInfo? {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        guard let netid = keychain["netid"] as? String, let password = keychain["password"] as? String else {
+            return nil
+        }
+        return (netid: netid, password: password)
+    }
+    
+}
+
+private let loginURL = URL(string: "https://get.cbord.com/cornell/full/login.php?mobileapp=1")
+private let maxTrials = 3
+private let trialDelay = 500
+
+// MARK: - Connection Handler
+
+private protocol BRBConnectionHandlerDelegate {
+
+    func brbConnectionHandlerDelegateDidRetrieve(sessionID: String)
+    func brbConnectionHandlerDelegateDidFailLogin(with error: String)
+
+}
+
+private class BRBConnectionHandler: WKWebView, WKNavigationDelegate {
+    
+    var stage: Stage = .loginScreen
+    private var diningHistory: [HistoryEntry] = []
+    private var loginCount = 0
+    var netid: String = ""
+    var password: String = ""
+    var delegate: BRBConnectionHandlerDelegate?
+    
```

Comment:
Put these in alphabetical order. Maybe do private vars on top in alphabetical order, then public variables right under also in alphabetical order:

ex. 

    private var a
    private var d

    var b
    var c